### PR TITLE
Bookin bar v2

### PR DESCRIPTION
**Pull request by Ysrbolles**
# Description

The New BookingBar component (v2), is Ready to use, but just on desktop, version mobile still needs some changes.
## Type of change

- [x]  New feature (non-breaking change which adds functionality)
- [x]  Add component
- [x]  Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Desktop
Screen Shot 2022-08-30 at 20 52 13

Screen Shot 2022-08-30 at 20 52 28

### Mobile

Screen Shot 2022-08-31 at 11 56 26

# Checklist:

- [x]  My code follows the style guidelines of this project
- [x]  I have performed a self-review of my code
- [x]  I have commented on my code, particularly in hard-to-understand areas
- [x]  I have made corresponding changes to the documentation
- [x]  My changes generate no new warnings
- [x]  I have added tests that prove my fix is effective or that my feature works
- [x]  New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/Bookings/BookingBar_v2/index.jsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}) {
+  const guestbtn = useRef(null) // ref for guests button
+  const { text, onClick } = buttonProps
+  const [isOpen, setIsOpen] = useState({
+    dates: false,
+    guests: false,
+  })
+
+  const onDatesChanges = async (dates) => {
+    await onDatesChange(dates)
```
Comment (redapy) [inline src/components/Bookings/BookingBar_v2/index.jsx:34]: can we change the name of one of these functions? They are kind of similar and you can't tell what's difference

File: src/components/Bookings/BookingBar_v2/index.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+          onClick={() => checkAvailability(dates, guestValues, baseUrl)}
+        >
+          {text}
+        </BookingButton>
+      </BookngContainer>
+    </Popover.Group>
+  )
+}
+
+BookingBa_v2.propTypes = {}
```
Comment (redapy) [inline src/components/Bookings/BookingBar_v2/index.jsx:117]: please add propTypes.

**Review by redapy — CHANGES_REQUESTED** [review]
Great work, there are small changes that I think they should be done.

File: src/components/Bookings/BookingBar_v2/index.jsx (hunk 1 above)
Comment (Ysrbolles) [reply src/components/Bookings/BookingBar_v2/index.jsx]: Fixed, thank you

**Comment by Ysrbolles** [comment]
Hello @redapy, I think you need to check the Buttons' size on each breakpoint.

Screen Shot 2022-08-31 at 11 56 26

Screen Shot 2022-08-31 at 11 36 31

**Comment by redapy** [comment]
@Ysrbolles The button is not taking a fixed width, it depends on its text. Maybe the designers should give reduce the `font-size` on mobile? I can also give less padding on mobile, but I don't think that's the problem

**Comment by Ysrbolles** [comment]
@redapy In this case, the button takes, an Icon not a text.

**Comment by redapy** [comment]
@Ysrbolles could you make an issue for it, please?

**Pull request merged**